package.json

```diff
@@ -25,7 +25,8 @@
 	"devDependencies": {
 		"@aws-sdk/abort-controller": "3.78.0",
 		"@aws-sdk/protocol-http": "3.78.0",
-		"@rivet-gg/api-internal": "0.0.1-c63067c",
+		"@rivet-gg/api": "https://github.com/rivet-gg/rivet/raw/3f21d8318fd8369dd129b59ac41107f537895b81/sdks/typescript/archive.tgz",
+		"@rivet-gg/api-ee": "file:vendor/rivet-gg-api-ee.tgz",
```

Comment:
Will this affect anyone developing without ee?

Reply:
Never mind, it seems that we are commiting the archive so that this is available 👍